lib/AppInfo/Application.php

```diff
@@ -32,10 +32,6 @@ public function __construct (array $urlParams=array()) {
 		/**
 		 * Core
 		 */
-		$container->registerService('UserId', function(IContainer $c) {
```

Comment:
why remove that service ?

Reply:
This function caused an error if the app gets accessed while logged out. Interestingly, the app works without a problem even if this container isn't registered, so I removed it. I re-added a fixed version if it is a problem that this container is missing.